packages/garbo/src/outfit/barf.ts

```diff
@@ -124,7 +124,8 @@ export function computeBarfOutfit(
     outfit.familiar === $familiar`Chest Mimic` &&
     $familiar`Chest Mimic`.experience < 550
   ) {
-    const famExpValue = (EMBEZZLER_MULTIPLIER() * get("valueOfAdventure")) / 50;
+    const famExpValue =
+      (MEAT_TARGET_MULTIPLIER() * get("valueOfAdventure")) / 50;
```

Comment:
Similarily, do we need something here in the case where the copy target isnt a meat target?

Reply:
I think for now this obeys the Hippocratic oath